src/main/java/seedu/duke/controllers/OrderController.java

```diff
@@ -0,0 +1,62 @@
+package seedu.duke.controllers;
+
+import seedu.duke.controllers.Controller;
+import seedu.duke.entities.Dish;
+import seedu.duke.manager.DishManager;
+import seedu.duke.manager.OrderManager;
+
+import java.util.Scanner;
+
+public class OrderController extends Controller {
+    private static final String[] CHOICES = {"0. Exit", "1. Display Menu",
+        "2. Create an order", "3. Delete an order",
+        "4. Get total value of current order",
+        "5. Get total value of all orders in the list", "6, Print receipt"
```

Comment:
Don't need to number choices because the parent class already number when printing out the choices

Reply:
I see